### PR TITLE
Community Article: Setting Up SFTP Users in RHEL 6, CentOS 6, Ubuntu …

### PR DESCRIPTION
**Pull request by BadCrumpany**
…& Debian: INFODEV-3247

### CODE REVIEW

**Comment by rackernexus** [comment]
Can one of the admins verify this patch?

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-c3514455c8/how-to/)

**Comment by RackerKate** [comment]
It doesn’t look like the step that begins at line 43 includes the actual line that should be commented out. Asked Shaun by email if he knows what the line should be.

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-0df1b15698/how-to/)

**Comment by RackerKate** [comment]
Sent questions to @BadCrumpany by email.

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-a9d0d2798e/how-to/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-54a9024c33/how-to/)

**Comment by RackerKate** [comment]
@cyrichardson Please review.

**Comment by cyrichardson** [comment]
@RackerKate On it.

**Comment by cyrichardson** [comment]
@RackerKate Suggested changes here: 
[set-up-sftp-users-in-rhel-centos-ubuntu-and-debian_cyrEdits.docx](https://github.com/rackerlabs/rackspace-how-to/files/2780536/set-up-sftp-users-in-rhel-centos-ubuntu-and-debian_cyrEdits.docx)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-ac12d9c7a8/how-to/). :bow:

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-7fce900b50/how-to/).

File: content/cloud-servers/setting-up-sftp-users-in-rhel-centos-ubuntu-debian.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+
+       sftp> pwd
+
+       Remote working directory: /html
+
+   SFTP only sees the files in the `/var/www/vhosts/mywebsite.com/` directory, and considers 
+   this directory the highest-level, root ('/') directory.
+
+7. Use the following steps to connect to SFTP and set up your SFTP client:
+
```
Comment (catlook) [inline content/cloud-servers/setting-up-sftp-users-in-rhel-centos-ubuntu-debian.md]: one gotcha: in the preview(https://staging.developer.rackspace.com/support.rackspace.com/build-7fce900b50/how-to/set-up-sftp-users-in-rhel-centos-ubuntu-and-debian/), steps a-g are wrapping and not on separate lines.

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-959a19aa32/how-to/). :bow:

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-8455f03331/how-to/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-663436073e/how-to/)

**Pull request merged**